src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -16,9 +16,19 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-"""
-STABLE.
-"""
+'''
+Provides a RadioToolButton class, similar to a "push" button.
+A group of RadioToolButtons can be set, so that only one can be
+selected at a time. When a button is clicked, it depresses and
+is shaded darker.
+
+It is also possible to set a tooltip to be dispalyed when the
+user scrolls over it with their cursor as well as an accelerator
+keyboard shortcut.
+
+Example:
+    .. literalinclude ../examples/radiotoolbutton.py
```

Comment:
Oh, this didn't build correctly.  Please test that in the future.

It should be:

```
Example:                                                                        
                                                                                
.. literalinclude:: ../examples/radiotoolbutton.py   
```